stdeb/util.py

```diff
@@ -257,13 +257,7 @@ def normstr(s):
 
 def get_date_822():
     """return output of 822-date command"""
-    cmd = '/bin/date'
-    if not os.path.exists(cmd):
-        raise ValueError('%s command does not exist.'%cmd)
-    args = [cmd,'-R']
-    result = get_cmd_stdout(args).strip()
-    result = normstr(result)
-    return result
+    return time.strftime("%a, %e %b %Y %H:%M:%S %z")
```

Comment:
Additionally `date -R` returns the name of the weekday / month in English - independent of the locale, whereas `%a` / `%b` are locale specific. Maybe use `list(calendar.day_abbr)` / `list(calendar.month_abbr)` instead?

Reply:
Fixed